.gitignore

```diff
@@ -18,8 +18,13 @@ bin
 
 !vendor/**/zz_generated.*
 
+# Mac file system
+.DS_Store
```

Comment:
Another PR has this. It also needs to be **/

Reply:
OK, will update.